src/components/ActionButton/index.ts

```diff
@@ -0,0 +1,26 @@
+import Block from '../../utils/Block';
+import tmpl from './ActionButton.hbs';
+import './ActionButton.scss';
+
+interface Props {
+    icon?: string;
+    title?: string;
+    modifier?: string;
```

Comment:
Здесь лучше использовать union вместо ```string```. При использовании ```string```, свойство может принять любое строковое значение, хотя ожидает ограниченный набор значений
 ```suggestion
    modifier?: 'delete' | 'update' | и тд;
```